src/main/java/org/jenkinsci/plugins/ghprb/GhprbRootAction.java

```diff
@@ -59,18 +59,20 @@ public void doIndex(StaplerRequest req, StaplerResponse resp) {
         String payload = null;
         String body = null;
 
-        if ("application/json".equals(type)) {
+        if (type.startsWith("application/json")) {
```

Comment:
Are you sure the string will always start with the application type?  Maybe a regex match would be more appropriate.